README.md

```diff
@@ -7,7 +7,7 @@ progress and are not ready for production use.
 
 ## Pre-requisites
 
-- Cert-manager installed and running
+- [Cert-manager](https://cert-manager.io/docs/installation) installed and running
```

Comment:
While we are at it, can we make this "cert-manager"? The project stylises itself like that.